src/components/Sender.tsx

```diff
@@ -81,23 +101,21 @@ const Sender = ({ className }: Props) => {
     const formatedAccounts = formatAccounts(accounts, ss58Format);
     const items = formatedAccounts.map(({ text, value, key }: any) => (
       <MenuItem
+        className={classes.selectAccountMainItem}
         key={key}
         value={value}
         onClick={() => {
           onChange(value, source);
         }}
       >
-        <div className={classes.row}>
-          <Account accountName={text} value={value} chain={source} />
-          <Account accountName={text} value={value} isDerived hideAddress chain={target} />
-        </div>
+        <Account friendlyName={text} value={value} chain={source} />
+        <Account friendlyName={text} value={value} chain={target} isDerived hideAddress />
       </MenuItem>
     ));
     return [
```

Comment:
Is there anyway to provide a better way in MUI to add headers to a group of options within the select? when i did this i felt it was kind of hacky but it was enough for putting something

Reply:
This is the reason for suggesting a lower Popover component: `// TODO replace MUI Select with MUI Popover it wraps around or Autocomplete to have more control over appearance`. I think it all will be nicer if we won't try to push all the logic in this Select

Reply:
Agree 100% , can you please create a GH issue and link the number in the TODO comment?